styles.css

```diff
@@ -1 +1,78 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+* {
+    padding: 0px;
+    margin: 0px;
+    background-color: #FFEFDB;
+
+
+}
+.grid {
```

Comment:
In de grid container komt de meeste code.
In de losse items (bijv. #a ,#b etc.) hoef je enkel de grid-area een naam te geven zodat je daar naartoe kan verwijzen.
Verder kan je de html elementen ipv alleen een ID ook een Class meegeven, zo kan je meerdere elementen met 1 blokje code stylen. Voorbeeld: `<div id="4" class="item">`  `.item{display: flex; }`. Zo hebben alle elementen met de Class item een display flex en hoef je dit niet per blokje aan te geven.